test/integration/scheduler_perf/dra/templates/resourceclaim-with-selector.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: resource.k8s.io/v1alpha3
+kind: ResourceClaim
+metadata:
+  name: test-claim-with-selector-{{.Index}}
+spec:
+  devices:
+    requests:
+    - name: req-0
+      deviceClassName: test-class
+      selectors:
+      - cel:
+          expression: |-
+            device.capacity['test-driver.cdi.k8s.io'].counters.compareTo(quantity('2')) >= 0 &&
+            device.attributes['test-driver.cdi.k8s.io'].preallocate == true
```

Does this simpler variant work?

```suggestion
            device.attributes['test-driver.cdi.k8s.io'].preallocate
```

---

Yeah, it does. I also noticed that we don't have any allocator tests that does selection based on attributes, so I created a PR to add one: https://github.com/kubernetes/kubernetes/pull/131849.